modules/cpmso-yaml-provider-spec-azure.adoc

```diff
@@ -0,0 +1,70 @@
+
+In the following example, `<cluster_id>` is the infrastructure ID that is based on the cluster ID that you set when you provisioned the cluster. If you have the OpenShift CLI installed, you can obtain the infrastructure ID by running the following command:
+
+[source,terminal]
+----
+$ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
+----
+
+.Sample Azure `providerSpec` values
+[source,yaml]
+----
+providerSpec:
+  value:
+    acceleratedNetworking: true
+    apiVersion: machine.openshift.io/v1beta1
+    credentialsSecret:
+      name: azure-cloud-credentials <1>
+      namespace: openshift-machine-api
+    diagnostics: {}
+    image: <2>
+      offer: ""
+      publisher: ""
+      resourceID: /resourceGroups/<cluster_id>-rg/providers/Microsoft.Compute/galleries/gallery_<cluster_id>/images/<cluster_id>-gen2/versions/412.86.20220930 <3>
+      sku: ""
+      version: ""
+    internalLoadBalancer: <cluster_id>-internal <4>
+    kind: AzureMachineProviderSpec <5>
+    location: <region> <6>
+    managedIdentity: <cluster_id>-identity
+    metadata:
+      creationTimestamp: null
+      name: <cluster_id>
+    networkResourceGroup: <cluster_id>-rg
+    osDisk: <7>
+      diskSettings: {}
+      diskSizeGB: 1024
+      managedDisk:
+        storageAccountType: Premium_LRS
+      osType: Linux
+    publicIP: false
+    publicLoadBalancer: <cluster_id> <8>
+    resourceGroup: <cluster_id>-rg
+    subnet: <cluster_id>-master-subnet <9>
+    subnet: {} <9>
+    userDataSecret:
+      name: master-user-data <10>
+    vmSize: Standard_D8s_v3
+    vnet: <cluster_id>-vnet
+    zone: {} <11>
+----
+<1> Specifies the secret name for the cluster. Do not change this value.
+<2> Specifies the image details for your control plane machine set.
+<3> Specifies an image that is compatible with your instance type. The Hyper-V generation V2 images created by the installation program have a `-gen2` suffix, while V1 images have the same name without the suffix.
+<4> Specifies the internal load balancer for the control plane. This field might not be prepopulated but is required in both the `ControlPlaneMachineSet` and control plane `Machine` CRs.
+<5> Specifies the cloud provider platform type. Do not change this value. 
+<6> Specifies the region to place control plane machines on.
+<7> Specifies the disk configuration for the control plane.
+<8> Specifies the public load balancer for the control plane.
+<9> #The example I have has `<cluster_id>-master-subnet`, but should it be `{}` and have this callout say "The subnet is configured in the failure domain, and is omitted here."?#
+<10> Specifies the control plane user data secret. Do not change this value.
+<11> #This line is not in the example I have, but should it be `{}` with "The availability zone is configured in the failure domain, and is omitted here."?#
```

Comment:
Can definitely just say "remove this" :)

Reply:
After chatting today, I have changed the annotation for all three things (AWSx2, Azurex1) that are specified in the failure domain to the following:
```
This parameter is configured in the failure domain, and is shown with an empty value here. If a value specified for this parameter differs from the value in the failure domain, the Operator overwrites it with the value in the failure domain.
```
LMK what you think about that, and if this is otherwise ok to send off for further review.

Reply:
That wording works for me

Reply:
Great, thanks! I am going to go ahead and comment out the items that I don't want to show up when this is merged, and send it along to QE